tests/test_conditional_flow_matcher.py

```diff
@@ -106,7 +107,14 @@ def test_fm(method, sigma, shape):
     x0, x1 = random_samples(shape, batch_size=batch_size)
     torch.manual_seed(TEST_SEED)
     np.random.seed(TEST_SEED)
-    t, xt, ut, eps = FM.sample_location_and_conditional_flow(x0, x1, return_noise=True)
+    eps = None
+    if test_eps:
+        eps = torch.randn_like(x0)
+    t, xt, ut, ret_eps = FM.sample_location_and_conditional_flow(
+        x0, x1, return_noise=True, eps=eps
+    )
+    if test_eps:
+        assert torch.allclose(ret_eps, eps)
```

Comment:
I think this assert should be done at the end of the file. eps is supposed to be the same as ret_eps and it is in the same spirit as the tests over t_given_init